### PR TITLE
Tests for feature extractors

### PR DESCRIPTION
**Pull request by Sidu28**
1) Moved import_info extractor to alphabetical features.
2) Added several new tests.

Also linked the "Add tests for remaining feature extractors" because I forgot to link that to the ngrams pull request I made a few days ago.

### CODE REVIEW

File: test/test_import_info.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from features.import_info import ImportInfoExtractor
+
+
+def test_disassemble():
+    extractor = ImportInfoExtractor('data/goodware_example')
+    assert (type(list(extractor.extract().values())[0]) == str)
+
+
+if __name__=='__main__':
+    test_disassemble()
```
Comment (mananshah99) [inline test/test_import_info.py:13]: Why are these lines necessary?

File: test/test_imported_exported_symbols.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import sys
+
+sys.path.append('../')
+from features.imported_symbols import ImportedSymbolsExtractor
+from features.exported_symbols import ExportedSymbolsExtractor
+
+
+def test_disassemble():
```
Comment (mananshah99) [inline test/test_imported_exported_symbols.py:8]: Why is this called test_disassemble and not test_imported_exported_symbols?

File: test/test_checksum.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import sys
+
+sys.path.append('../')
+from features.checksum import ChecksumExtractor
+
+
+def test_disassemble():
```
Comment (mananshah99) [inline test/test_checksum.py:7]: Why is this called test_disassemble?

File: test/test_import_info.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import sys
+
+sys.path.append('../')
+from features.import_info import ImportInfoExtractor
+
+
+def test_disassemble():
```
Comment (mananshah99) [inline test/test_import_info.py:7]: Why is this called test_disassemble?

File: test/test_section_info.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import sys
+
+sys.path.append('../')
+from features.section_info import SectionInfoExtractor
+
+
+def test_disassemble():
```
Comment (mananshah99) [inline test/test_section_info.py:7]: Why is this called test_disassemble?

**Review by mananshah99 — CHANGES_REQUESTED** [review]
Super minor changes, rest looks good

**Comment by Sidu28** [comment]
Whoops, forgot to change the function names.  All done, I'll go ahead and merge
